examples/pytorch/nlp/huggingface_models/text-classification/pruning/eager/run_glue_no_trainer_new_api.py

```diff
@@ -0,0 +1,635 @@
+# coding=utf-8
```

Comment:
@chensuyue  please check the license usage

Reply:
If we modify the code, add one copyright `# Copyright (c) 2023 Intel Corporation`